static/coffee/flows/controllers.coffee

```diff
@@ -1056,6 +1056,10 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
     ruleset.rules[0].category[Flow.flow.base_language] = 'All Responses'
 
   formData.timeoutOptions = [
+    {value:0.166667, text:'10 seconds'},
+    {value:0.333333, text:'20 seconds'},
```

Comment:
There's something going wrong on the Old Flow Editor because I'm getting this error when I refresh the page and try to do some change on the flow, seems that the value is a string instead of a float number when refreshing the page


![Nov-26-2019 17-09-40](https://user-images.githubusercontent.com/5198870/69668988-b9a16d80-106f-11ea-8357-4f9fa395ad0d.gif)

Reply:
make sure you are using the updated version of mailroom. It needs to be built.

Reply:
fix [here](https://github.com/greatnonprofits-nfp/rapidpro/pull/29/commits/12c5cd28a022a2d90114f4a579cded3d56b610d5)